tensorflow/lite/micro/kernels/PORTING.md

```diff
@@ -0,0 +1,251 @@
+# Porting Reference Ops from Lite to Micro
+
```

Comment:
might be worth a TOC using gh-md-toc similar to https://github.com/tensorflow/tensorflow/blob/faf9a3386c46fe1b18686e551abc21e4b2f10547/tensorflow/lite/micro/CONTRIBUTING.md#L3-L6